apps/mediator/src/push-notifications/fcm/services/PushNotificationsFcmService.ts

```diff
@@ -78,4 +76,16 @@ export class PushNotificationsFcmService {
       await this.pushNotificationsFcmRepository.save(agentContext, pushNotificationsFcmRecord)
     }
   }
+
+  public async getPushNotificationRecordByConnectionId(agentContext: AgentContext, connectionId: string) {
+    const pushNotificationsFcmRecord = await this.pushNotificationsFcmRepository.getSingleByQuery(agentContext, {
+      connectionId,
+    })
+
+    if (!pushNotificationsFcmRecord) {
```

Comment:
Get single by query already throws if not found, so i think this doesn't do anything